docs/api.rst

```diff
@@ -12032,3 +12032,55 @@ Add EVM Transaction By Hash
    :statuscode 409: No user is currently logged in.
    :statuscode 500: Internal rotki error.
    :statuscode 502: An external service used in the query such as etherscan could not be reached or returned unexpected response.
+
+
+Get Binance Savings Balance Details
+=======================================
+
+.. http:get:: /api/(version)/(location)/savings
+
+   Doing a GET on this endpoint will return an object containing the amount in the savings balance and the interest earned.
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``.
+
+   .. note::
+      ``"location"`` can only be one of ``"binance"`` or ``"binanceus"``.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/binance/savings HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"async_query": true}
+
+   :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not.
```

Comment:
Include the location argument and the description. The note above should be an entry here instead